src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -28,10 +28,10 @@ object LocalBackend {
 class LocalBackend extends Backend {
 
   /**
-   * Executes the specified command line, using the supplied call and symbol store for expression evaluation.
+   * Executes the specified command line, using the supplied lookup function for expression evaluation.
    * Returns a `Map[String, Try[WdlValue]]` of output names to values.
    */
-  override def executeCommand(commandLine: String, call: Call, taskOutputs: Seq[TaskOutput], symbolStore: SymbolStore): Map[String, Try[WdlValue]] = {
+  override def executeCommand(commandLine: String, call: Call, taskOutputs: Seq[TaskOutput], scopedLookupFunction: ScopedLookupFunction): Map[String, Try[WdlValue]] = {
```

Comment:
This is way off topic for this PR but I was looking at the `executeCommand` implementation and I had a few thoughts:

1) Run command using `/bin/bash -c "... escaped command ..."` so it's easier to log standard out or to a log file and it makes one less temp file.

2) It's not checking return codes.